application/controllers/programmes.php

```diff
@@ -335,8 +337,8 @@ protected function diff_revisions($year, $type, $programme_id = false, $revision
 	 * @param string $type         The type of programme, either ug or pg (not currently used).
 	 * @param int    $revision_id  The ID of the revision being submitted for editing.
 	 */
-	public function get_submit_programme_for_editing($year, $type, $object_id, $revision_id)
-	{	
+	public function post_submit_programme_for_editing($year, $type, $object_id, $revision_id)
```

Comment:
changed this controller route method to be POST instead of GET because HTTP... plus we are now posting the changelog and material change indicator to it